Signal/translations/en.lproj/Localizable.strings

```diff
@@ -389,7 +389,7 @@
 "CREATE_NEW_GROUP" = "Create new group";
 
 /* Title for the 'crop/scale image' dialog. */
-"CROP_SCALE_IMAGE_VIEW_TITLE" = "Crop Image";
+"CROP_SCALE_IMAGE_VIEW_TITLE" = "Crop and Scale";
```

Comment:
I don't think most users know what "Scale" means.  I prefer the old title.

Reply:
Both WhatsApp and iOS System contacts editing go with "Move and Scale" so I'm actually now inclined to go this third way.